troposphere/static/js/components/modals/volume/VolumeCreateModal.react.js

```diff
@@ -281,6 +288,7 @@ define(
               <h4>Projected Resource Usage</h4>
               {this.renderStorageConsumption(identity, size, volumes)}
               {this.renderStorageCountConsumption(identity, size, volumes)}
+              <a className="modal-link" href="#" onClick={this.handleResourceRequest}>Need more resources{String.fromCharCode(63)}</a>
```

Comment:
You can replace String.fromCharCode(63) with question mark in quotes like this:

{"Need more resources?"}